src/pages/playing-for-keeps-terms.md

```diff
@@ -0,0 +1,145 @@
+---
+template: legal-page
+path: /playing-for-keeps-terms
+title: Playing for Keeps Terms of Use
+date: Last Updated April 12, 2020
+---
+# Rules & Rewards
+
+Keep is committed to helping you succeed on this program and to building a robust decentralized network. To this end, they've set up a [Discord server](https://discord.gg/wYezN7v) that anyone can join, and [technical documentation](https://docs.keep.network/run-random-beacon.html) to help you run Keep's random beacon client and start staking on testnet.
+
+## **Reward Guidelines**
+
+## Current Game of Keeps
+
+The current game of Keeps begins on April 15th.
+
+## Rewards
+
+Judges will determine what the reward status for each entry is, in their sole discretion. Programs that are suggested by the judges will have their reward status posted.
+
+*   Level 1: Up to 100,000 Keep
+*   Level 2: Up to 10,000 Keep
+*   Level 3: Up to 1,000 Keep
+*   Level 4: Up to 500 Keep
+*   Level 5: No Keep
+
+In addition to severity, other variables are also considered when the judges decide the score, including (but not limited to):
+
+*   User Demand. Higher rewards are paid for projects that will be demanded by the users.
+*   Quality of description. Higher rewards are paid for clear, well-written submissions.
+*   Quality of reproducibility. Please include test code, scripts and detailed instructions. The easier it is for us to reproduce and verify the vulnerability, the higher the reward. Please see the docs and repos to learn more about our test suite in the official [documentation](https://docs.keep.network/run-random-beacon.html).
```

Comment:
/cc @mhluongo it also looks like we may want to update this when we finalize bug bounty info, given it's mentioned here.